draft-ietf-ppm-dap.md

```diff
@@ -898,6 +896,13 @@ The parameters pertaining to specific batch modes are described in
 
 ### Time-interval Batch Mode {#time-interval-batch-mode}
 
+~~~ tls-presentation
```

Comment:
Do we really need to split each value out into its own section? IMO, this is harder to follow than what we had before.

I think we could move these back to where they were (e.g. centralize the definition of hte `BatchMode` enum), while still using these sections as the reference point for the Batch Modes Registry section, without loss of readability. (Is there a convention / requirement that the numeric code point be defined in the section referenced by the IANA table?) Alternatively, we could move these back to where they were & have the Batch Modes Registry section refer back to the Batch Modes section for all defined batch modes.

Reply:
Part of my goal is to make these sections self-contained in a way that makes life a little easier for folks writing documents that specify new batch modes. Such documents will always have a line like this:

```
enum {
   cool_guy(42), (255)
} BatchMode;
```

and now that line can be copy-pasted from our RFC.

I also think this is a little simpler for whoever at IANA is making sure they get the registry right.

To answer your questiona bout conventions: I'm not aware of a hard and fast rule here. However this is a common pattern for example in TLS: https://tlswg.org/draft-ietf-tls-esni/draft-ietf-tls-esni.html#section-5-2